specification/search/data-plane/Azure.Search/preview/2020-06-30-Preview/searchindex.json

```diff
@@ -992,6 +1085,54 @@
       "additionalProperties": true,
       "description": "A single bucket of a facet query result. Reports the number of documents with a field value falling within a particular range or having a particular value or interval."
     },
+    "AnswerResult": {
+      "properties": {
+        "score": {
+          "type": "integer",
+          "format": "int64",
+          "readOnly": true,
+          "description": "The score value representis how relevant the answer is to the the query relative to other answers returned for the query."
+        },
+        "key": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The key of the document the answer was extracted from.",
+          "x-ms-client-name": "Key"
+        },
+        "text": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The text passage extracted from the document contents as the answer.",
+          "x-ms-client-name": "Text"
+        },
+        "highlights": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Same text passage as in the Text property with highlighed text phrases most relevant to the query.",
+          "x-ms-client-name": "Text"
+        }
+      },
+      "additionalProperties": true,
+      "description": "An answer is a text passage extracted from the contents of the most relevant documents that matched the query. Answers are extracted from the top 50 search results. The answer candidates are scored and the top answers are selected."
```

Comment:
I'm not sure "50 documents" is true. I think we only consider the top 10 passages. Maybe we don't need to give a number, and instead just say the "top documents"